psi-ms.obo

```diff
@@ -24760,6 +24760,156 @@ relationship: has_units UO:0000191 ! fraction
+id: MS:4000190
+name:  MS1 TIC quantile RT
+def: "The amount of time elapsed since the start of the data acquisition until the implied quantile of all MS1 TIC has been accumulated. The number of values in the tuple implies the quantile mode." [PSI:MS]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_metric_category MS:4000016 ! retention time metric
+relationship: has_metric_category MS:4000021 ! MS1 metric
+relationship: has_units UO:0000031 ! minute
+relationship: has_value_type xsd:float
+
+[Term]
+id: MS:4000191
+name:  MS2 TIC quantile RT
+def: "The amount of time elapsed since the start of the data acquisition until the implied quantile of all MS2 TIC has been accumulated. The number of values in the tuple implies the quantile mode." [PSI:MS]
+is_a: MS:4000004 ! n-tuple
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_metric_category MS:4000016 ! retention time metric
+relationship: has_metric_category MS:4000022 ! MS2 metric
+relationship: has_units UO:0000031 ! minute
+relationship: has_value_type xsd:float
+
+[Term]
+id: MS:4000192
+name: MS1 median cycle time
+def: "The median time between consecutive MS1 scans across the full run." [PSI:MS]
+synonym: "sampling frequency" RELATED [MS:1000029]
+synonym: "fastest frequency for MS level 1 collection" RELATED [MS:4000065]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_metric_category MS:4000019 ! MS metric
+relationship: has_units UO:0000010 ! second
+relationship: has_value_type xsd:float
+
+[Term]
+id: MS:4000193
+name: DIA isolation window median cycle time
+def: "The median time between consecutive measurements of the same DIA isolation window across the full run." [PSI:MS]
+synonym: "sampling frequency" RELATED [MS:1000029]
+synonym: "fastest frequency for MS level 2 collection" RELATED [MS:4000066]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_metric_category MS:4000019 ! MS metric
+relationship: has_units UO:0000010 ! second
+relationship: has_value_type xsd:float
+
+[Term]
+id: MS:4000194
+name: DIA isolation window count
+def: "The number of distinct DIA isolation windows (i.e. sharing the same low m/z boundary, high m/z boundary, and ion mobility parameters)." [PSI:MS]
+is_a: MS:4000003 ! single value
+relationship: has_metric_category MS:4000009 ! ID free metric
+relationship: has_metric_category MS:4000012 ! single run based metric
+relationship: has_metric_category MS:4000022 ! MS2 metric
+relationship: has_metric_category MS:1000792 ! isolation window attribute
```

Comment:
This is distinct from `is_a: MS:1000792` so it is not legal to use in an actual isolation window? This is purely curiosity.

Reply:
It's intended as a QC metric, so the relationship is just meant to add some more semantics. I didn't think about a use case for having it as an attribute for an actual isolation window as well. But I'm not sure it makes that much sense, because this is aggregated information across all isolation windows, rather than a property of a single isolation window.